### DISCUSSION TITLE
Add support for the custom gravity system

### DISCUSSION THREAD

**Discussion by victorhmrod** [Sixze/ALS-Refactored#571] [Ideas]
Add support for the custom gravity system! Because by adding the functionality the character does not rotate around the gravitational object
![imagem_2024-10-15_215807349](https://github.com/user-attachments/assets/1fc4131a-a891-47e6-ab2f-631e4dcaf947)

**Comment by ZundleFire** [comment]
I've been trying to figure this out. Have you found a solution for this? 

The animations don't update to the new gravity direction vector. Can't seem to find where the animations down/up vector is calculated or set. The third person template animgraph is easy to update to the new gravity.
 
Followed the UE custom gravity guide.
